src/lib/controller/controllerState.ts

```diff
@@ -1,4 +1,4 @@
-import controllerPub from '../../roslib/controllerPub';
+import { useRoslibStore } from '@/store/useRoslib';
```

Comment:
i cant comment on the line, but be consistent whether you want export on the same line as the declaring `export class Foo {}` or on a new line `class Foo; export { Foo }`

Reply:
Changed all classes to `export default Foo` 👍

Reply:
Let me clarify, cuz I still see a few instances of this in the code. Some code uses the single line syntax to export, seen here: https://github.com/TrickfireRobotics/mission-control/blob/625c3f2ebc8ef0f8cccedd6e6eebb21dbd919235/src/roslib/heartbeatPub.ts#L6

Some places define what they want to export, then export it on a separate line, like so: https://github.com/TrickfireRobotics/mission-control/blob/625c3f2ebc8ef0f8cccedd6e6eebb21dbd919235/src/script/saveCSVData.ts#L58

You should choose one and stick to it, for consistency.